modules/telco-upgrade-upgrading-the-worker-nodes.adoc

```diff
@@ -0,0 +1,130 @@
+// Module included in the following assemblies:
+//
+// * edge_computing/day_2_core_cnf_clusters/upgrading/telco-upgrade-completing-the-upgrade.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="telco-upgrade-upgrading-the-worker-nodes_{context}"]
+= Upgrading the worker nodes
+
+You upgrade the worker nodes after you have upgraded the control plane by unpausing the relevant `MCP` groups you created.
+Unpausing the `MCP` group starts the upgrade process for the worker nodes in that group.
+Each of the worker nodes in the cluster reboot to upgrade to the new EUS, y-stream or z-stream version as required.
+
+In the case of EUS to EUS upgrades note that when a worker node is upgraded it will only require one reboot and will jump <y+2>-release versions. This is a feature that was added to decrease the amount of time that it takes to upgrade large bare-metal clusters.
+
+[IMPORTANT]
+====
+This is a potential holding spot. Which means that a cluster is fully supported to run in production with the control plane that is upgraded to a new EUS release while the worker nodes are at at <y-2>-release. This allows large clusters to upgrade in steps across several maintenance windows.
+====
+
+[NOTE]
```

Comment:
Can this note be moved further down in the procedure, nearer to the MCP steps, so that it can avoid the back-to-back admonitions here?